src/server.cpp

```diff
@@ -2760,16 +2665,20 @@ void Server::sendDetachedInventory(const std::string &name, session_t peer_id)
 
 void Server::sendDetachedInventories(session_t peer_id, bool incremental)
 {
-	for (const auto &detached_inventory : m_detached_inventories) {
-		const std::string &name = detached_inventory.first;
-		if (incremental) {
-			Inventory *inv = detached_inventory.second;
-			if (!inv || !inv->checkModified())
-				continue;
+	// Lookup player name, to filter detached inventories just after
+	std::string peer_name;
+	if (peer_id != PEER_ID_INEXISTENT) {
+		RemotePlayer *player = m_env->getPlayer(peer_id);
+		if (player) {
```

Comment:
should return if player does not exist

Reply:
yep exact, i'll add a warning too in case ot